src/transform/ElementUtilities.js

```diff
@@ -44,6 +46,7 @@ const isNestedInTable = (el) => {
 }
 
 export default {
+  matchesSelectorCompat,
```

Comment:
Oh could we use a `test` container here?
<pre>
export default {
  findClosestAncestor,
  isNestedInTable,
  test: {
    matchesSelectorCompat
  }
}
</pre>